bolt-ktor/pom.xml

```diff
@@ -5,18 +5,17 @@
     <parent>
         <groupId>com.slack.api</groupId>
         <artifactId>slack-sdk-parent</artifactId>
-        <version>1.45.4-SNAPSHOT</version>
+        <version>1.46.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>bolt-ktor</artifactId>
-    <version>1.45.4-SNAPSHOT</version>
+    <version>1.46.0-SNAPSHOT</version>
     <packaging>jar</packaging>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <kotlin.code.style>official</kotlin.code.style>
-        <!-- TODO: upgrade to 3.x -->
-        <ktor.version>2.3.12</ktor.version>
+        <ktor.version>3.1.3</ktor.version>
```

Comment:
Just changing the version does not work, and that's the reason why this task was not done then